src/uiTest/resources/ideprobe.conf

```diff
@@ -0,0 +1,30 @@
+probe {
+  driver {
+    vmOptions = [
+      # TODO (JetBrains/intellij-ui-test-robot#191): turn off DEBUG logs from Ktor & Netty completely
```

Comment:
Pls +1 this issue! not linking explicitly here so that our [astroturfing](https://pl.wikipedia.org/wiki/Astroturfing) isn't too obvious